sdk/include/sdk/DataPointReply.h

```diff
@@ -46,24 +46,33 @@ class DataPointReply final {
     DataPointReply(DataPointMap_t&& dataPointsMap)
         : m_dataPointsMap(std::move(dataPointsMap)) {}
 
+    /**
+     * @brief Get the desired data point from the reply as generic type.
+     *
+     * @param path The path ("name") of the data point to query from the reply.
+     * @return std::shared_ptr<DataPointValue>  The genric data point value contained in the reply.
+     */
+    [[nodiscard]] std::shared_ptr<DataPointValue> getGeneric(const std::string& path) const {
```

Comment:
```suggestion
    [[nodiscard]] std::shared_ptr<DataPointValue> getGeneric(const std::string& path) const {
```
Consider `getUntyped` as name. `getGeneric` sounds like it is dealing with C++'s generics which it is not.